RubeesEat/wwwroot/css/site.css

```diff
@@ -113,6 +110,193 @@ nav > ul > li:hover {
     transform: rotate(-45deg) translate(7px, -6px);
 }
 
+p, h1, h2, h3, h4, h5, h6 {
+    overflow-wrap: break-word;
+    text-align: center;
+    margin-top: 0;
+    margin-bottom: 2rem;
+    font-weight: 300;
+}
+
+p { font-size: 1.0rem; line-height: 1.2;  letter-spacing: -.1rem;}
+h1 { font-size: 2.5rem; line-height: 1.2;  letter-spacing: -.1rem;}
+h2 { font-size: 2.5rem; line-height: 1.25; letter-spacing: -.1rem; }
+h3 { font-size: 2.0rem; line-height: 1.3;  letter-spacing: -.1rem; }
+h4 { font-size: 1.5rem; line-height: 1.35; letter-spacing: -.08rem; }
+h5 { font-size: 1.0rem; line-height: 1.5;  letter-spacing: -.05rem; }
+h6 { font-size: 0.5rem; line-height: 1.6;  letter-spacing: 0; }
+
+/* Forms
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+input[type="number"],
+input[type="text"],
+select {
```

Comment:
The `select` element has bug where the layout changes when the element is focused vs. not focused.

Reply:
I don't see the bug when I try it myself. Weirdly enough, when I try to set the border and outline in the normal select selector, and then only change the color when focused (so like this):

select {
height: 38px;
padding: 6px 10px; /* The 6px vertically centers text on FF, ignored by Webkit */
background-color: white;
border-radius: 4px;
border: 1px solid;
outline: 0;
max-width: 200px;
}

input[type="number"]:focus,
input[type="text"]:focus,
select:focus {
border: var(--primary-color);
}

THEN i suddenly get a bug where the text of the select element starts moving around when I click on it and off it, even though I only want to change the color. But before I didn't have that bug

Reply:
Turns out it was already fixed when discussing it during standup